.tool-versions

```diff
@@ -0,0 +1,3 @@
+ruby 3.2.2
+nodejs 18.17.0
+yarn 1.22.19
```

Comment:
@pablobm Oh I forgot to pick the removal of these files when merging your branch. I think we can specify a Node version in `package.json`